Mathlib/Logic/UnivLE.lean

```diff
@@ -24,30 +21,42 @@ noncomputable section
 /--
 A class expressing a universe inequality. `UnivLE.{u, v}` expresses that `u ≤ v`.
 
-There are (at least) two plausible definitions for `u ≤ v`:
-* strong: `∀ α : Type max u v, Small.{v} α`
-* weak: `∀ α : Type u, Small.{v} α`
-
-The weak definition has the advantage of being transitive.
-However only under the strong definition do we have `Small.{v} ((α : Type u) → (β : Type v))`,
-which is essential for proving that `Type v` has `Type u`-indexed limits when `u ≤ v`.
+There used to be a stronger definition `∀ α : Type max u v, Small.{v} α` that immediately implies
+`Small.{v} ((α : Type u) → (β : Type v))` which is essential for proving that `Type v` has
+`Type u`-indexed limits when `u ≤ v`. However the current weaker condition
+`∀ α : Type u, Small.{v} α` also implies the same, so we switched to use it for
+its simplicity and transitivity.
```

Comment:
Am I right in saying that the strong condition is now spelt `UnivLE.{max u v,v}`?

Reply:
Yes, exactly. See [this file](https://github.com/leanprover-community/mathlib4/pull/8556/files#diff-1f1d21773565ef19790bcc2e45b860cf458ed91c502f7c835e8e741530c2a94d).

Reply:
Can you mention that in the docstring? My reading was that it seemed to suggest that the strong version was no longer expressible using the contents of this file.